sql/size_utils.sql

```diff
@@ -67,46 +67,25 @@ RETURNS TABLE (
 AS '@MODULE_PATHNAME@', 'ts_dist_remote_chunk_info' LANGUAGE C VOLATILE STRICT;
 
 CREATE OR REPLACE FUNCTION _timescaledb_internal.hypertable_local_size(
-    schema_name_in name,
-    table_name_in name)
+	schema_name_in name,
+	table_name_in name)
```

Comment:
Why do we use schema + table name here but regclass in `hypertable_detailed_size`? Shouldn't we stick to the same conventions in both tables?

Reply:
This is an internal function and since we don't have regclass in our internal metadata, this is what our internal functions work with. Otherwise, I think a larger refactoring requires a larger scope than the issue this PR is trying to fix.